nautobot/core/api/serializers.py

```diff
@@ -100,6 +101,22 @@ def fields(self):
         return self.__pruned_fields
 
 
+@extend_schema_field(
+    {
+        "type": "object",
+        "properties": {
+            "id": {
+                "type": "string",
+                "format": "uuid",
+            },
+            "object_type": {"type": "string"},
```

Comment:
Non-blocking, but could consider adding a `pattern` regex to the `object_type` schema for greater precision.